examples/tools/bevymark.rs

```diff
@@ -72,67 +75,74 @@ fn scheduled_spawner(
             scheduled.per_wave,
             bird_texture.0.clone_weak(),
         );
-        counter.color = Color::rgb_linear(random(), random(), random());
+
+        let mut random = thread_rng();
```

Comment:
Because this is example code, could you please comment to explain why this is faster.

Reply:
You mean here or as a comment in the code? `thread_rng` uses an `Rc` under the hood so theoretically we save some instructions by not cloning it a bunch of times.

Reply:
Something like
```
// cache the rng
```
Might be enough enough here.